person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @age = age
+    @name = name
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    age.of_age? || parent_permission
+  end
```

Comment:
- [ ] We should correctly use the `of_age?` method we defined.

- [ ] We should also use the `@parent_permission` instance variable, not `parent_permission` as this would throw errors.

  ```
  def can_use_services?
    of_age? || @parent_permission
  end
  ```